molecule/default/playbook.yml

```diff
@@ -5,14 +5,6 @@
     pulp_config_secret_key: secret
     pulp_default_admin_password: ''
     ansible_python_interpreter: /usr/bin/python3
-  pre_tasks:
-    - name: Require Ansible 2.4+
```

Comment:
This is being dropped because the roles declare a minimum Ansible version, right?

Reply:
true